### DISCUSSION TITLE
May there be a problem using Wine?

### DISCUSSION THREAD

**Discussion by liweiyang2023** [prasanthrangan/hyprdots#877] [Q&A]
When I use Wine to compile, there is a black box blocking the Wine interface. Every time I click, the black box will be regenerated and covered on the interface. I don't know if it's a Wine issue or something

**Comment by prasanthrangan** [comment]
Hi, sorry i dont understand. Could you please share screenshot? and any error message?

**Comment by liweiyang2023** [comment]
> Hi, sorry i dont understand. Could you please share screenshot? and any error message?

![IMG_20231004_140003.jpg](https://github.com/prasanthrangan/hyprdots/assets/144606759/5c09f6d6-70b1-4c26-8189-c78f86159abf)

![IMG_20231004_140008.jpg](https://github.com/prasanthrangan/hyprdots/assets/144606759/558b912b-efa6-4198-a5eb-c981371440da)

He looks like this. He can only be displayed by dragging the black screen to zoom out, but after zooming out, clicking on it will be covered by a black cloth. I didn't encounter this problem with kde before, so I think it's a layer stacking order issue?

**Comment by prasanthrangan** [comment]
> He looks like this. He can only be displayed by dragging the black screen to zoom out, but after zooming out, clicking on it will be covered by a black cloth. I didn't encounter this problem with kde before, so I think it's a layer stacking order issue?

Is this running on xwayland?? kde has its own `xdg-desktop-portal` so might be an issue with `xdg-desktop-portal-hyprland`

**Comment by TheyCallMeSecond** [comment]
im using wine with bottles and i dont have any problem

**Comment by kRHYME7** [comment]
No configurations related to wine so maybe not. I'll be moving this to discussion to segregate, issues. 🥂
